mlflow/tracking/_model_registry/utils.py

```diff
@@ -93,13 +94,20 @@ def set_registry_uri(uri: str) -> None:
     _registry_uri = uri
 
 
+def _get_registry_uri_from_spark_session():
+    if (session := _get_active_spark_session()) is None:
```

Comment:
Maybe this instead? Just for better readability
```suggestion
    session = _get_active_spark_session()
    if session is None:
```